pallets/allocations/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "pallet-allocations"
-version = "2.4.2"
```

Comment:
What would the version of our own pallet?

Reply:
All versions are read from the workspace. One place to update all versions.